blocks/progress-bar-table/progress-bar-table.js

```diff
@@ -0,0 +1,85 @@
+function calculateRatioPost(elements) {
```

Comment:
- use speaking names for variables (e.g. `numElements` instead of `sizel`
- observe camel case
- don't overwrite variables that are not the same conceptually (e.g. `ratio1`)
- reduce lines of code

for example: 
```
function calculateRatioPost(elements) {
  const numElements = elements.length;

  if (numElements === 2) {
    return [100, 0];
  }

  const childNodes1 = elements[1].childNodes;
  const childNodes2 = elements[2].childNodes;

  const progress1 = parseFloat(childNodes1[7].textContent.substring(0, childNodes1[7].textContent.indexOf(' ')));
  const progress2 = parseFloat(childNodes2[7].textContent.substring(0, childNodes1[7].textContent.indexOf(' ')));

  let ratio1 = 100;
  let ratio2 = 100;

  if (progress2 > progress1) {
    ratio1 = Math.floor((progress1 * 100) / progress2);
  } else {
    ratio2 = Math.floor((progress2 * 100) / progress1);
  }

  return [ratio1, ratio2];
}
```